src/context/AuthContext.jsx

```diff
@@ -0,0 +1,9 @@
+import { createContext, useState } from 'react';
+
+export const AuthContext = createContext();
+
+export const AuthProvider = ({ children }) => {
+	const [userInfo, setUserData] = useState([{}, {}]);
```

Comment:
@Srish-ty, if a user is already logged in the data should be populated here I believe. Can use the `onAuthStateChanged` or some similar function to achieve this.

Reply:
added onAuthStateChanged. https://github.com/dscnitrourkela/project-udon/pull/11/commits/c798e968ffc0ed1a9916de3aa4e019061e231665